server/test/repository/ProgramRepositoryTest.java

```diff
@@ -380,88 +379,8 @@ public void getApplicationsForAllProgramVersions_searchById() {
     assertThat(paginationResult.getNumPages()).isEqualTo(1);
   }
 
-  private static ImmutableList<Object[]> getSearchByNameOrEmailData() {
```

Comment:
@nb1701 I think you might also be able to remove the `@RunWith(JUnitParamsRunner.class)` annotation on this test class as well.